### PR TITLE
Update middleware.js

### PR DESCRIPTION
**Pull request by cexoso**
fix error TypeError: Cannot set property 'webpackStats' of undefined

### CODE REVIEW

File: lib/middleware.js [hunk 1]
```diff
@@ -14,6 +14,7 @@ module.exports = function wrapper(context) {
 
       return new Promise(((resolve) => {
         ready(context, () => {
+          res.locals = res.locals || {};
```
Comment (ev1stensberg) [inline lib/middleware.js]: Can you link to any issue regarding this or fill a longer description of what you're trying to fix? Would help us understand your problem space, as this issue might be deeper than just setting a default. Thanks! ❤️

**Comment by cexoso** [comment]
#282

**Comment by shellscape** [comment]
PR template removed. Closing as invalid.

**Comment by shellscape** [comment]
Well, I was going to reopen this and merge it but the CLA wasn't signed so we can't. @cexoso I'm going to implement your fix and give you credit in the commit message. In the future it's important not to delete or alter issue and pull request templates.

**Pull request closed without merging**